packages/clerk-js/src/core/devBrowserHandler.ts

```diff
@@ -135,6 +135,8 @@ export default function createDevBrowserHandler({
       isCookielessDev = true;
       const data = await resp.json();
       setDevBrowserJWT(data?.token);
+    } else {
```

Comment:
We could be more strict here and also check the response status code (`400`) and the error code (`cookieless_dev_disabled`)